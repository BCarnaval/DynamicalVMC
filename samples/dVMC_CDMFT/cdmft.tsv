model     	omega     	E_kin       	E_pot     	U 	eb1_1     	eb2_1      	eb3_1     	eb4_1      	mu 	t 	tb1_1      	tb2_1      	tb3_1      	tb4_1      	ave_mu     	ave_t       	ave_U_1 	ave_eb1_1 	ave_eb2_1 	ave_eb3_1 	ave_eb4_1 	ave_mu_1   	ave_t_1     	ave_tb1_1 	ave_tb2_1 	ave_tb3_1 	ave_tb4_1 	E0_1       	sector_1   	n_1        	githash 	version 	GS_consistency 	iterations 	dist_function     	distance 	diff_hybrid 	omegaH    	script             	time             	
2L_4b_gen 	2.3444509 	-0.97360941 	2.5990834 	4 	1.1395773 	-1.1138588 	1.1396978 	-1.1138494 	2  	1 	0.43173701 	0.43189134 	0.43176232 	0.431895   	1.0005549  	-0.97360941 	nan     	nan       	nan       	nan       	nan       	1.0000909  	-0.68023998 	nan       	nan       	nan       	nan       	-9.7603015 	R0:N6:S0:1 	1.0000909  	20a4d9d 	v2.1.5  	Y              	28         	sharp_wc_2.0_b_25 	2.47e-04 	4.98e-03    	2.3444509 	general_bath_1D.py 	2023-07-14@12:20 
model     	omega     	E_kin       	E_pot     	U 	eb1_1     	eb2_1      	eb3_1     	eb4_1      	mu 	t 	tb1_1      	tb2_1      	tb3_1      	tb4_1      	ave_mu     	ave_t       	ave_U_1 	ave_eb1_1 	ave_eb2_1 	ave_eb3_1 	ave_eb4_1 	ave_mu_1   	ave_t_1     	ave_tb1_1 	ave_tb2_1 	ave_tb3_1 	ave_tb4_1 	E0_1       	sector_1   	n_1        	githash 	version 	GS_consistency 	iterations 	dist_function     	distance 	diff_hybrid 	omegaH    	script             	time             	
2L_4b_gen 	2.3254649 	-0.99380719 	2.69314   	4 	1.1852808 	-1.1052508 	1.1853073 	-1.105313  	2  	1 	0.43284244 	0.43097885 	0.43284955 	0.43099256 	0.99255218 	-0.99380719 	nan     	nan       	nan       	nan       	nan       	0.99508838 	-0.69917342 	nan       	nan       	nan       	nan       	-9.7370343 	R0:N6:S0:1 	0.99508838 	20a4d9d 	v2.1.5  	Y              	26         	sharp_wc_2.0_b_25 	2.10e-04 	7.33e-03    	2.3254649 	general_bath_1D.py 	2023-07-17@10:45 
