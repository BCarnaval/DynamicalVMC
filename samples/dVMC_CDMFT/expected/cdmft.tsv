model     	omega     	E_kin       	E_pot     	U 	eb1_1     	eb2_1      	eb3_1     	eb4_1      	mu 	t 	tb1_1      	tb2_1      	tb3_1      	tb4_1      	ave_mu     	ave_t       	ave_U_1 	ave_eb1_1 	ave_eb2_1 	ave_eb3_1 	ave_eb4_1 	ave_mu_1   	ave_t_1     	ave_tb1_1 	ave_tb2_1 	ave_tb3_1 	ave_tb4_1 	E0_1       	sector_1   	n_1        	githash 	version 	GS_consistency 	iterations 	dist_function     	distance 	diff_hybrid 	omegaH    	script             	time             	
2L_4b_gen 	2.3254649 	-0.99380719 	2.69314   	4 	1.1852808 	-1.1052508 	1.1853073 	-1.105313  	2  	1 	0.43284244 	0.43097885 	0.43284955 	0.43099256 	0.99255218 	-0.99380719 	nan     	nan       	nan       	nan       	nan       	0.99508838 	-0.69917342 	nan       	nan       	nan       	nan       	-9.7370343 	R0:N6:S0:1 	0.99508838 	20a4d9d 	v2.1.5  	Y              	26         	sharp_wc_2.0_b_25 	2.10e-04 	7.33e-03    	2.3254649 	general_bath_1D.py 	2023-07-17@10:45 
